packages/select-rich/src/LionSelectRich.js

```diff
@@ -247,11 +247,6 @@ export class LionSelectRich extends SlotMixin(ScopedElementsMixin(OverlayMixin(L
     this._onFormElementsChanged();
   }
 
-  // TODO: move to overlayMixin and offer open and close
-  toggle() {
```

Comment:
Yeah toggle is already in OverlayMixin so no need to override it here in LionSelectRich and therefore not a breaking change to remove this.